src/components/Tray/index.tsx

```diff
@@ -45,11 +53,39 @@ export const Tray: FC<{
       }}
     >
       <LittleTitle titleInCorner={notInHome || newsOpen} />
+      <Pressable
+        style={{
+          height: 20,
+          backgroundColor: "blue",
+          marginRight: 10,
+        }}
+        onPress={async () => {
+          await Notifications.scheduleNotificationAsync({
+            content: { title: "prova", body: "Prova", data: {} },
+            trigger: null,
+          })
+        }}
+      >
+        <BodyText>Notif</BodyText>
+      </Pressable>
+      <Pressable
+        style={{
+          height: 20,
+          backgroundColor: "red",
+          marginRight: 10,
+        }}
+        onPress={() => {
+          notificationCentre.clearStorage()
+        }}
+      >
+        <BodyText>Clear</BodyText>
+      </Pressable>
```

Comment:
direi che sta roba per testare può scomparire adesso, virtualmente se ce n'è bisogno possiamo creare un menù di debug da qualche parte nelle impostazioni ahah

Reply:
Ok, la funzione cleanup in fondo a NotificationCenter la rimuovo giusto? tanto non viene mai chiamata

Reply:
uhm sì effettivamente non sembra molto utile, vero che non da nessun fastidio stando lì però boh non vedo perché si dovrebbe mai voler togliere i listener